shared/constants/platform.native.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import {Dimensions, Platform, NativeModules} from 'react-native'
 import {cachesDirectoryPath} from '../util/file.native'
+import * as iPhoneXHelper from 'react-native-iphone-x-helper'
```

Comment:
this was a dep that i needed to pull in for something that i removed but the code is tiny and more specific than ours so keeping it